modules/rosa-sts-ocm-roles-and-permissions.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// rosa_architecture/rosa-sts-understanding-roles-and-permissions.adoc
+//
+:_content-type: PROCEDURE
+[id="rosa-sts-ocm-roles-and-permissions_{context}"]
+= {cluster-manager} roles and permissions
+
+When creating clusters, you must have at least three roles to create and use the clusters. These roles are:
+
+* A user role
+* A basic `ocm-role` resource
+* An admin `ocm-role` resource
+
+[id="rosa-sts-ocm-roles-and-permissions-iam-user-role_{context}"]
+== User role
+
+The user role has no additional permissions. {cluster-manager} identifies this user role and creates trust between this role, the Red Hat account, and the two `ocm-role` resources that you create.
+
+//Used to identify the user by creating a trust between it, Red Hat account and the ocm role.
+
+[id="rosa-sts-ocm-roles-and-permissions-iam-basic-role_{context}"]
+== Creating basic {cluster-manager} permissions
+
+The basic {cluster-manager} role permissions allow you to perform cluster maintenance within {cluster-manager}.
+
+.Procedure
+* Create a basic `ocm-role` resource:
```

Comment:
You could update this to state something like:

"* Create an `ocm-role` IAM role with basic privileges:"